android/app/src/main/java/org/bitcoinppl/cove/flows/SelectedWalletFlow/ReceiveAddressSheet.kt

```
@@ -146,10 +147,32 @@ private fun ReceiveAddressSheetContent(
+    LaunchedEffect(addressGeneratedTime) {
+        while(true) {
+            if (addressGeneratedTime == 0L) {
+                timeLeftStr = ""
+            } else {
+                val elapsed = System.currentTimeMillis() - addressGeneratedTime
+                val remaining = WalletManager.WaitTimeToGenerateNewAddress - elapsed
+                
+                if (remaining > 0) {
+                    val minutes = (remaining / 1000) / 60
+                    val seconds = (remaining / 1000) % 60
+                    timeLeftStr = String.format("Auto-refresh in %02d:%02d", minutes, seconds)
```

detekt (1.23.8)

[warning] String.format("Auto-refresh in %02d:%02d", minutes, seconds) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)